classes/step0_form.php

```diff
@@ -46,6 +46,7 @@ public function definition () {
         $mform->setExpanded('manualhdr', true);
 
         $plugintypes = tool_pluginskel\local\util\manager::get_plugintype_names();
+        asort($plugintypes);
```

Comment:
Is this what you mean? I wonder if there are places in core Moodle that could also benefit from this change.
```suggestion
        \core_collator::asort($plugintypes);
```

Reply:
> Is this what you mean?

Yup, that.

> in core Moodle

See my comment above - this was discussed before and there are pros and cons.

Reply:
> > in core Moodle
> 
> See my comment above - this was discussed before and there are pros and cons.

Of course! I was not recommending that we "always sort alphabetically", but that places where raw `asort()` is already being used on arrays whose values are user-facing strings may want to consider the core_collator's asort() instead. Example: https://github.com/moodle/moodle/blob/511a87f5fc357f18a4c53911f6e6c7f7b526246e/mod/lesson/editpage_form.php#L49

Reply:
Oh yes of course. Well spotted!